django_cas_ng/views.py

```diff
@@ -109,8 +109,9 @@ def login(request, next_page=None, required=False):
         elif settings.CAS_RETRY_LOGIN or required:
             return HttpResponseRedirect(_login_url(service))
         else:
-            error = "<h1>Forbidden</h1><p>Login failed.</p>"
-            return HttpResponseForbidden(error)
+            #error = "<h1>Forbidden</h1><p>Login failed.</p>"
+            #return HttpResponseForbidden(error)
```

Comment:
Can you remove these commented lines.  No need to leave this code here.  That's what version control is for.